pkg/network/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -99,24 +111,114 @@ func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) (types.Result, error) {
 	return result, nil
 }
 
-// Send the ADD command environment and config to the CNI server, printing
-// the IPAM result to stdout when called as a CNI plugin
-func (p *cniPlugin) skelCmdAdd(args *skel.CmdArgs) error {
-	result, err := p.CmdAdd(args)
+func (p *cniPlugin) testCmdAdd(args *skel.CmdArgs) (types.Result, error) {
+	return p.doCNIServerAdd(newCNIRequest(args), "dummy0")
+}
+
+func (p *cniPlugin) CmdAdd(args *skel.CmdArgs) error {
+	req := newCNIRequest(args)
+	ifname := req.Env["CNI_IFNAME"]
```

Comment:
args.IfName and args.Netns.  But you can ignore the "" check if you want as these are already checked for you in the 'skel' package.